### PR TITLE
update apache proposal

### PR DESCRIPTION
**Pull request by neverchanje**

### CODE REVIEW

File: _community/proposal.md [hunk 1]
```diff
@@ -238,10 +238,15 @@ The existing code already has unit tests so we will make use of existing Apache
 
 ## Source and Intellectual Property Submission Plan
 
-Current code is Apache 2.0 licensed and the copyright is assigned to XiaoMi.
+Most of the current code is Apache 2.0 licensed and the copyright is assigned to XiaoMi.
 If the project enters incubator, XiaoMi will transfer the source code & trademark
 ownership to ASF via a Software Grant Agreement.
 
+But due to historical issues, Pegasus was based on MIT licensed codes that were initially
```
Comment (levy5307) [inline _community/proposal.md]: code is mass noun. Here is a [disscussion](https://english.stackexchange.com/questions/20455/is-it-wrong-to-use-the-word-codes-in-a-programming-context) about it.

**Comment by acelyc111** [comment]
Add PacificA paper link https://www.microsoft.com/en-us/research/wp-content/uploads/2008/02/tr-2008-25.pdf for "PacificA" in line 17.

**Comment by acelyc111** [comment]
Add "and is expensive to provide scalability" after line 32.

**Comment by acelyc111** [comment]
Line 94: baidu ->Baidu

**Comment by acelyc111** [comment]
> 70+ deployments, 800+ applications (about 70+TB)

Would "70+ clusters, 800+ tables (about 70+TB), xxx servers" be more clear?

**Comment by acelyc111** [comment]
> The Pegasus project is in production use already inside XiaoMi, but is not expected to be an Baidu product for external customers.

What dose "an Baidu product" mean?

**Comment by acelyc111** [comment]
> Client libraries with different Languages:

languages

**Comment by acelyc111** [comment]
> and it is heavily refactored and developed to be more fit in Pegasus.

fit for?

**Comment by acelyc111** [comment]
Some other external dependencies not mentioned:
- concurrentqueue
- readerwriterqueue
- fds
- prometheus-cpp
- curl
- nlohmann_json

**Comment by acelyc111** [comment]
And some source code we use directly:
- linenoise

**Comment by neverchanje** [comment]
> Add "and is expensive to provide scalability" after line 32.

I just want to emphasize that Pegasus is designed to support the persistency that Redis does not have. We don't need to fully compare the two systems. I'm afraid it will raise unnecessary debates if we state that Pegasus is more scalable.

**Review by acelyc111 — APPROVED** [review]

**Pull request merged**